spring-cloud-dataflow-server-core/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This allows us to mock the final class `WebSecurity` to validate the customizer sets the proper ignore paths.